pandas/core/window/ewm.py

```diff
@@ -129,6 +129,7 @@ def __init__(
         adjust=True,
         ignore_na=False,
         axis=0,
+        **kwargs,
```

Comment:
This is just for compatibility and so I don't think it needs documenting, please let me know if this is wrong